drivers/filesystems/cdfs/volinfo.c

```diff
@@ -160,6 +160,8 @@ Return Value:
         //  and false if it couldn't wait for any I/O to complete.
         //
 
+        RtlZeroMemory(Irp->AssociatedIrp.SystemBuffer, Length);
```

Comment:
Please, wrap this change with `#ifdef __REACTOS__`

Reply:
Thank you, my mistake :D